packages/e2e/cypress/e2e/api/projectPermissions.cy.ts

```diff
@@ -236,7 +253,8 @@ describe('Lightdash API tests for member user with admin project permissions', (
 
 describe('Lightdash API tests for member user with editor project permissions', () => {
     let email;
-    beforeEach(() => {
+
+    before(() => {
```

Comment:
This will only create 1 user per `describe` test, then we can simply login before each teast. This improves speed (and reduces users created) by 10x